FaceTracking/ViewController.swift

```diff
@@ -141,11 +150,12 @@ extension ViewController: AVCaptureVideoDataOutputSampleBufferDelegate {
         
         for feature in features {
             if let faceFeature = feature as? CIFaceFeature {
+                
                 let faceRect = calculateFaceRect(facePosition: faceFeature.mouthPosition, faceBounds: faceFeature.bounds, clearAperture: cleanAperture)
                 let featureDetails = ["has smile: \(faceFeature.hasSmile)",
-                    "has closed left eye: \(faceFeature.leftEyeClosed)",
-                    "has closed right eye: \(faceFeature.rightEyeClosed)"]
-                
+                    "has closed right eye: \(faceFeature.leftEyeClosed)",
```

Comment:
are you sure it's ok?

Reply:
No. I will set it back to the original orientation.